twentytwentyone-blocks/block-templates/404.html

```diff
@@ -0,0 +1,25 @@
+<!-- wp:template-part {"slug":"header","theme":"twentytwentyone-blocks","align":"full", "tagName":"header","className":"site-header"} /-->
+
+<!-- wp:heading {"level":1,"align":"wide"} -->
+<h1 class="alignwide">Nothing Here</h1>
```

Comment:
Once translation is possible in these templates, these strings should be adjusted so they're translatable.